shared/test/views_test.js

```diff
@@ -758,7 +753,18 @@ describe("loop.shared.views", function() {
 
       expect(element).not.eql(null);
       expect(element.className).eql("local-video");
-      expect(element.muted).eql(true);
+
+      // Google Chrome doesn't seem to set "muted" on the element at creation
+      // time, so we need to do the test as async.
```

Comment:
Weird.  I wonder if this is really React papering over some difference in Chrome.